NAMESPACE

```diff
@@ -7,7 +7,6 @@ export(abline)
 export(arrows)
 export(axis)
 export(bgCol)
-export(bgCol.default)
```

Comment:
OH...but then you don't know what the arguments are in the help files....I get it now. So....bgCol is unique because the only required variable is col. I think I'd actually vote to put it back in so we can include those arguments in the help files.